src/SFA.DAS.ApprenticeFeedback.Domain.UnitTests/Entities/ApprenticeFeedbackTargetTests.cs

```diff
@@ -212,6 +213,35 @@ public void WhenCalling_ResetFeedbackTarget_SetsDefaultsOnTarget(Domain.Entities
             }
         }
 
+        [Test, RecursiveMoqAutoData]
```

Comment:
What does `RecursiveMoqAutoData` do ?

Reply:
The MoqAutoData uses a builder to create examples for parameters the Recursive would do so recursively